experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -193,8 +193,8 @@ export class SharedPropertyTree extends SharedObject {
 	}
 
 	private scopeFutureDeltasToPaths(paths?: string[]) {
-		const socket = (this.runtime.deltaManager as any).deltaManager.connectionManager.connection
-			.socket;
+		const socket = (this.runtime.deltaManager as any).deltaManager.deltaManager
+			.connectionManager.connection.socket;
```

Comment:
Created https://dev.azure.com/fluidframework/internal/_workitems/edit/3553 for this.

Reply:
What about backcompat.  Don't you need to check both depth levels?

Reply:
`while(deltaManager.deltaManager !== undefined) { deltaManager = deltaManager.deltaManager; }`  😹